hwy/ops/emu128-inl.h

```diff
@@ -109,6 +109,37 @@ HWY_API VFromD<D> BitCast(D /* tag */, VFrom v) {
   return to;
 }
 
+// ------------------------------ ResizeBitCast
+
+template <class D, class VFrom>
+HWY_API VFromD<D> ResizeBitCast(D d, VFrom v) {
+  using DFrom = DFromV<VFrom>;
+  using TFrom = TFromD<DFrom>;
+  using TTo = TFromD<D>;
+
+  constexpr size_t kFromByteLen = sizeof(TFrom) * HWY_MAX_LANES_D(DFrom);
```

Comment:
Note that Simd<> has a MaxBytes() member function which could simplify these. Do you want to update the code? (It's also fine as-is, just more verbose.)